cmd/influxd/launcher/launcher.go

```diff
@@ -897,7 +898,8 @@ func (m *Launcher) run(ctx context.Context, opts *InfluxdOpts) (err error) {
 		)
 	}
 
-	notebookServer := notebookTransport.NewNotebookHandler(m.log.With(zap.String("handler", "notebooks")))
+	notebookSvc, _ := notebookSvc.NewFakeStore()
```

Comment:
Even though the error is always `nil` for now, can we check it here? That way we don't forget about it later when we replace `NewFakeStore` with something real.

Reply:
Yep, good idea. I'll have to think about what kind of errors (if any?) could be encountered at this point when I do the real implementation. I've changed it to explicitly log the error and return if one is encountered.